### PR TITLE
@FIR-204: Revised and sanitized changes from SSD

### PR DESCRIPTION
**Pull request by atrivedi-tsavoritesi**
Removed following changes from patch
1. Enable PCI-E root port in DTS
2. Remove forcing of Gen 2 in DTS
3. Remove forcing of 4 io queues
4. Remove increased timeouts in nvme code

@mmankal I have only kept code which is absolutely needed. We can look at config flags as well but right now all of the above changes are not needed in the code.

### CODE REVIEW

**Review by mmankal — COMMENTED** [review]
I wanted to multiple optimizations.  I  want to try out these and  until then please do not merge this code to main branch

**Review by akapoor3518 — APPROVED** [review]
lgtm

**Review by mmankal — COMMENTED** [review]
All the code that was in these files were necessary changes that were needed for troubleshooting and debug of the NVME driver for Sandisk 1TB M.2 . With this code the driver worked just fine. However, in the interest of reducing the code footprint and to keeping the code as close as possble to the main linux-socfpga - some of these chnages were revised and sanitized.

**Pull request merged**
